Vegetarian-Food-Finder/Controller/RecommendViewController.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  RecommendViewController.swift
+//  Vegetarian-Food-Finder
+//
+//  Created by Abdullahi Ahmed on 7/20/22.
+//
+
+import UIKit
+
+class RecommendViewController: UIViewController {
+    let segueFromRecommendToDetails = "segueFromRecommendToDetails"
+    var restaurantsList: [Business] = []
+    var selectedRestaurant: RestaurantScore!
```

Comment:
RestaurantScore is holding a full instance of a restaurant, duplicating the source of truth and taking unnecessary space